Public/Assistants/Set-Assistant.ps1

```diff
@@ -97,7 +97,7 @@ function Set-Assistant {
 
         # Invoke base function
         $steppablePipeline = {
-            PSOpenAI\New-Assistant @Parameters
+            PSOpenAI\New-Assistant @paramseters

```

Comment:
There is a change that appears to be a batch replacement error. The same change has been made to many files. Please explain the reason for the change or fix it.

Reply:
Oh jeez, my apologies. the next PR will be better and have actual code 🙏🏼